data_schema_struct.go

```diff
@@ -0,0 +1,2517 @@
+					Mode string `json:"mode,omitempty"`
+					// Electrical energy lost by power dissipation to heat inside the AC/DC converter.
+					PowerLoss float64 `json:"powerLoss,omitempty"`
+					// Start or stop the charging process.
+					StartStopCharging string `json:"startStopCharging,omitempty"`
+					// Current temperature of AC/DC converter converting grid voltage to battery voltage.
+					Temperature float64 `json:"temperature,omitempty"`
+					// The time needed for the current charging process to reach Charging.ChargeLimit. 0 if charging is complete or no charging process is active or planned.
+					TimeToComplete uint32 `json:"timeToComplete,omitempty"`
+					// Properties related to timing of battery charging sessions.
+					Timer struct {
+						// Defines timer mode for charging: INACTIVE - no timer set, charging may start as soon as battery is connected to a charger. START_TIME - charging shall start at Charging.Timer.Time. END_TIME - charging shall be finished (reach Charging.ChargeLimit) at Charging.Timer.Time. When charging is completed the vehicle shall change mode to 'inactive' or set a new Charging.Timer.Time. Charging shall start immediately if mode is 'starttime' or 'endtime' and Charging.Timer.Time is a time in the past.
+						Mode string `json:"mode,omitempty"`
+						// Time for next charging-related action, formatted according to ISO 8601 with UTC time zone. Value has no significance if Charging.Timer.Mode is 'inactive'.
+						Time string `json:"time,omitempty"`
+					} `json:"timer,omitempty"`
+				} `json:"charging,omitempty"`
+				// Current current flowing in/out of battery. Positive = Current flowing in to battery, e.g. during charging. Negative = Current flowing out of battery, e.g. during driving.
+				CurrentCurrent float64 `json:"currentCurrent,omitempty"`
+				// Current electrical energy flowing in/out of battery. Positive = Energy flowing in to battery, e.g. during charging. Negative = Energy flowing out of battery, e.g. during driving.
+				CurrentPower float64 `json:"currentPower,omitempty"`
+				// Current Voltage of the battery.
+				CurrentVoltage float64 `json:"currentVoltage,omitempty"`
+				// Properties related to DC/DC converter converting high voltage (from high voltage battery) to vehicle low voltage (supply voltage, typically 12 Volts).
+				DCDC struct {
+					// Electrical energy lost by power dissipation to heat inside DC/DC converter.
+					PowerLoss float64 `json:"powerLoss,omitempty"`
+					// Current temperature of DC/DC converter converting battery high voltage to vehicle low voltage (typically 12 Volts).
+					Temperature float64 `json:"temperature,omitempty"`
+				} `json:"dCDC,omitempty"`
+				// Gross capacity of the battery.
+				GrossCapacity uint16 `json:"grossCapacity,omitempty"`
+				// Battery Identification Number as assigned by OEM.
+				Id string `json:"id,omitempty"`
```

Comment:
The linter is complaining about this but I think the gRPC generator does the same thing.